skyvern/forge/sdk/api/llm/config_registry.py

```diff
@@ -37,7 +37,17 @@ def register_config(cls, llm_key: str, config: LLMRouterConfig | LLMConfig) -> N
     @classmethod
     def get_config(cls, llm_key: str) -> LLMRouterConfig | LLMConfig:
         if llm_key not in cls._configs:
-            raise InvalidLLMConfigError(llm_key)
+            # If the key is not found in registered configs, treat it as a general model
+            LOG.info("Using general model configuration for unknown LLM key", llm_key=llm_key)
+            if not settings.LLM_KEY:
+                raise InvalidLLMConfigError(f"LLM_KEY not set for {llm_key}")
+            return LLMConfig(
+                llm_key,  # Use the LLM_KEY as the model name
```

Comment:
do you mean settings.LLM_KEY here?

```suggestion
                settings.LLM_KEY,  # Use the LLM_KEY as the model name
```

Reply:
It's the same value I think

Reply:
llm_key is not in the config. How would this even work?

Reply:
what is an example of llm_key?

OPENAI_GPT4_1, OPENAI_GPT4_1_MINI and OPENAI_GPT4_5 would be a valid llm_key.

Are you trying to support llm_key like `gpt-4.1`?

Reply:
yeah exactly. If people pass in 'gpt-4.1` under LLM_KEY today, they would be able to use gpt-4.1

I find it really annoying to keep updating this whenever new models get released! Consider this a v0 towards generalization :)

Reply:
Got it. in this case, if liltellm supports this model, then it works? i'm not sure if it'll work tho but worth a try

Reply:
would be great if we can have some documentations abt it